### DISCUSSION TITLE
Submitted and verified account from valid college address yet not approved

### DISCUSSION THREAD

**Discussion by ArijitDutta1** [community/community#124857] [GitHub Education]
Hi, I have verified my university email addresses and also submitted my id proof from my college location, yet I have not been approved, can someone please help

**Comment by nikhilsbuilds** [comment]
Hi, I'm here to help you, 
Visit here - https://education.github.com/discount_requests/application here you can check your verification status.
and if you are already verified, check - https://education.github.com/pack/offers here for the Student's Benefits.
